wicket-core/src/main/java/org/apache/wicket/core/util/string/JavaScriptUtils.java

```diff
@@ -151,6 +152,30 @@ public static void writeJavaScriptUrl(final Response response, final CharSequenc
 	 */
 	public static void writeJavaScriptUrl(final Response response, final CharSequence url,
 		final String id, boolean defer, String charset, boolean async)
+	{
+                writeJavaScriptUrl(response, url, id, defer, charset, async, null);
+	}
+	/**
+	 * Write a reference to a javascript file to the response object
+	 * 
+	 * @param response
+	 *            The HTTP response
+	 * @param url
+	 *            The javascript file URL
+	 * @param id
+	 *            Unique identifier of element
+	 * @param defer
+	 *            specifies that the execution of a script should be deferred (delayed) until after
+	 *            the page has been loaded.
+	 * @param charset
+	 *            a non null value specifies the charset attribute of the script tag
+	 * @param async
+	 *            specifies that the script can be loaded asynchronously by the browser
+         * @param integrity
+         *            integrity attributes
+	 */
+	public static void writeJavaScriptUrl(final Response response, final CharSequence url,
+		final String id, boolean defer, String charset, boolean async, final IntegrityAttributed integrity)
```

Comment:
Perhaps it's better to switch to an attribute map instead? The method had too much parameters already.